project/Dependencies.scala

```diff
@@ -11,14 +11,15 @@ object Dependencies {
   val PlayJsonVersion = "2.6.7"
   val PlayStandaloneWsVersion = "1.1.3"
   val TwirlVersion = "1.3.12"
-  val PlayFileWatchVersion = "1.0.0"
+  val PlayFileWatchVersionForSbt013 = "1.0.0"
+  val PlayFileWatchVersionForSbt10 = "1.1.5"
```

Comment:
We could probably just use the same play-file-watch version for both sbt 1 and 0.13.